geth/transactions/utils/utils.go

```diff
@@ -1,4 +1,4 @@
-package common
+package utils
```

Comment:
This file contains methods used in test files, for example, `ImportTestAccount` and in regular files. It would be beneficial to split it.